tests/functional/test_related_sc_data/bundle_configs/A_X_to_A_Y/cases/case_1.yaml

```diff
@@ -0,0 +1,6 @@
+description: Attempt to save HC map with only A.X on 1 host
+bound_to_raise_exception:
+requires_raise_exception:
```

Comment:
I don't think that this is the correct way of using YAML keys. Either pass them as list values or remove them at all and use existing logics from bound components tests.